src/components/scenes/LoginScene.tsx

```diff
@@ -178,6 +182,11 @@ export function LoginSceneComponent(props: Props) {
     dispatch(showSendLogsModal()).catch(err => showError(err))
   })
 
+  // Wait for the sticky config to initialize before rendering anything
+  useAsyncEffect(async () => {
+    setLegacyLanding((await getStickyConfigValue('legacyLanding')) as boolean)
```

Comment:
No need for the type assertion if `getStickyConfigValue` returns the correct type.